core/kernel/persistence/Cache.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2023 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\generis\model\kernel\persistence;
+
+use oat\oatbox\cache\SimpleCache;
+
+interface Cache
```

Comment:
For me this interface is weird, not really a cache, since it just knows how to return a real cache. Maybe the property could know if it `supportsCache` by checking whether Model config has cache configured?

Reply:
If we call it `Cacheable`?